samplePDF/samplePDFFDBase.cpp

```diff
@@ -13,24 +13,15 @@ samplePDFFDBase::samplePDFFDBase(double pot, std::string mc_version, covarianceX
   //ETA - safety feature so you can't pass a NULL xsec_cov
   if(xsec_cov == NULL){std::cerr << "[ERROR:] You've passed me a NULL xsec covariance matrix... I need this to setup splines!" << std::endl; throw;}
 
-  init(pot, mc_version, xsec_cov);          
-
-  bNu = new BargerPropagator();
-  bNu->UseMassEigenstates(false);
-  bNu->SetOneMassScaleMode(false);
-  bNu->SetWarningSuppression(true);
-  
-  Beta=1;
-  useBeta=false;
-  applyBetaNue=false;
-  applyBetaDiag=false;
-
   samplePDFFD_array = NULL;
   samplePDFFD_data = NULL;
   
   //KS: For now FD support only one sample
   nSamples = 1;
   SampleName.push_back("FDsample");
+
+  //ETA - make sure osc is NULL!!
+  //Osc = NULL; 
```

Comment:
why is  this commented?

Reply:
I removed this. It's set to NULL in samplePDFFDBase.h